src/update_mhq_aq_hr2025/2_admin_aq_watersurfaces.Rmd

```diff
@@ -304,4 +304,333 @@ assessments_update %>%
+
+check_missing_inboveg <-  measurements_cycle1_inboveg %>%
+    filter(!is.na(type)) %>%
+    filter(is.na(recording_givid)) %>%
+    anti_join(measurements_cycle1_match, by = c("measurement_date", "polygon_id")) %>%
+    left_join(popunits, by = c("polygon_id", "type")) %>%
+    select(polygon_id, db_ref, type, measurement_year, assessment_date)
+    
+```
+
+These are the resulting measurements in cycle 1.
+
+```{r}
+measurements_cycle1_match %>%
+    datatable(rownames = FALSE,
+              filter = "top")
+```
+
+For the following measurements no INBOVEG record is found.
+
+```{r}
+check_missing_inboveg %>%
+    datatable(rownames = FALSE,
+              filter = "top")
+```
+
+
+### Measurements cycle2
+
+We first check for which measurements we find a record in INBOVEG for the same `polygon_id`, `type` and `measurement_date`.
+
+```{r}
+measurements_cycle2_match <- measurements_cycle2 %>%
+    inner_join(ws_inboveg, by = c("polygon_id", "type", "measurement_date")) %>%
+    filter(suitable_mhq) %>%
+    mutate(mhq_cycle = 2)
+```
+
+```{r}
+measurements_cycle2_match %>%
+    select(polygon_id, measurement_date, type, suitable_mhq, recording_givid) %>%
+    datatable(rownames = FALSE,
+              filter = "top") 
+```
+Next we check mismatches in `measurement_date`.
+
+```{r}
+check_date <- measurements_cycle2 %>%
+    anti_join(measurements_cycle2_match, by = c("polygon_id", "type", "measurement_date")) %>%
+    inner_join(ws_inboveg, by = c("polygon_id", "type", "measurement_year"), suffix = c("_admin", "_inboveg")) %>%
+    select(polygon_id, type, measurement_year, starts_with("measurement_date"))
+
+check_date %>%
+    datatable(rownames = FALSE)
+```
+
+```{r}
+check_type <- measurements_cycle2 %>%
+    anti_join(measurements_cycle2_match, by = c("polygon_id", "type", "measurement_date")) %>%
+    inner_join(ws_inboveg, by = c("polygon_id", "measurement_date"), suffix = c("_admin", "_inboveg")) %>%
+    select(polygon_id, starts_with("type"), starts_with("measurement_date"))
+```
+
+Following table gives an overview of the measurements not imported in INBOVEG yet.
+
+```{r}
+check_missing_inboveg <- measurements_cycle2 %>%
+    anti_join(measurements_cycle2_match, by = c("polygon_id", "type", "measurement_date")) %>%
+    anti_join(check_date, by =  c("polygon_id", "type", "measurement_year"))
+
+check_missing_inboveg %>%
+    select(polygon_id, type, measurement_date, in_inboveg) %>%
+    datatable(rownames = FALSE)
+```
+
+
+```{r}
+mhq_watersurfaces_cycle2_sf <- st_read("../../data/design_cycle2/mhq_standingwater_cycle2_2024-04-17.gpkg")
+
+mhq_watersurfaces_cycle2 <- mhq_watersurfaces_cycle2_sf %>%
+    st_drop_geometry() %>%
+    # separate_longer_delim(cols = type_all, delim = "; ") %>%
+    select(polygon_id, grts_ranking, grts_ranking_draw, type_all, area_class, sac, panel)
+
+watersurfaces <- read_watersurfaces()
+watersurfaces_hab <- read_watersurfaces_hab()
```

Comment:
Best be explicit about versions here (e.g. look at checksums).

Reply:
I added documentationon the versions.